### PR TITLE
[MINOR] Fix flaky testes in HealthCheckCoordinatorGrpcTest

### PR DESCRIPTION
**Pull request by frankliee**
### What changes were proposed in this pull request?
Increase timeout setting (3 -> 5)

### Why are the changes needed?
Fix flaky testes in HealthCheckCoordinatorGrpcTest

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
UT

### CODE REVIEW

File: integration-test/common/src/test/java/com/tencent/rss/test/HealthCheckCoordinatorGrpcTest.java [hunk 1]
```diff
@@ -102,7 +102,7 @@ public void healthCheckTest() throws Exception {
           1,
           1,
           Sets.newHashSet(Constants.SHUFFLE_SERVER_VERSION));
-    Uninterruptibles.sleepUninterruptibly(3, TimeUnit.SECONDS);
+    Uninterruptibles.sleepUninterruptibly(5, TimeUnit.SECONDS);
```
Comment (colinmjj) [inline integration-test/common/src/test/java/com/tencent/rss/test/HealthCheckCoordinatorGrpcTest.java:105]: is it possible to replace sleep with CoordinatorTestUtils.waitForRegister(coordinatorClient,2)?

Comment (jerqi) [reply integration-test/common/src/test/java/com/tencent/rss/test/HealthCheckCoordinatorGrpcTest.java:105]: It's not useful. I try to add `CoordinatorTestUtils.waitForRegister`. But this test is still flaky.

Comment (colinmjj) [reply integration-test/common/src/test/java/com/tencent/rss/test/HealthCheckCoordinatorGrpcTest.java:105]: here is to test if coordinator has 2 shuffle server
how about using `CoordinatorTestUtils.waitForRegister` to `replace assertEquals(2, coordinatorClient.getShuffleServerList().getServersCount())`;

Comment (jerqi) [reply integration-test/common/src/test/java/com/tencent/rss/test/HealthCheckCoordinatorGrpcTest.java:105]: It's not flaky place.

Comment (jerqi) [reply integration-test/common/src/test/java/com/tencent/rss/test/HealthCheckCoordinatorGrpcTest.java:105]: We should sleep enough time to receive  the unheathly  heatbeat .

Comment (colinmjj) [reply integration-test/common/src/test/java/com/tencent/rss/test/HealthCheckCoordinatorGrpcTest.java:105]: got it

**Review by jerqi — APPROVED** [review]

**Pull request merged**